pdns/dnsdist-cache.cc

```diff
@@ -302,14 +302,14 @@ size_t DNSDistPacketCache::purgeExpired(size_t upTo, const time_t now)
   const size_t maxPerShard = upTo / d_shardCount;
 
   size_t removed = 0;
-  size_t scannedMaps = 0;
+  uint32_t shardIndex = 0;
 
   do {
```

Comment:
Not wrong, but now that `d_expungeIndex` is gone you might as well turn this into a regular `for (range)` loop